Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -74,7 +74,7 @@ public States States
         /// </summary>
         [HideInInspector]
         [SerializeField]
-        private int InputActionId = -1;
+        private int InputActionId = 0;
```

Comment:
Do you happen to know why this serializes an `int` instead of the actual `MixedRealityInputAction`?

Reply:
Because that is the way it was originally implemented. InputAction, both in Interactable and in MRTK, has a lot of issues. Existing item for here is: https://github.com/microsoft/MixedRealityToolkit-Unity/issues/6105